.github/actions/setup/action.yml

```diff
@@ -0,0 +1,48 @@
+name: Setup node and pnpm
+description: Setup node and install dependencies using pnpm
+inputs:
+  use_lockfile:
+    description: 'Whether to use the lockfile vs latest floating dependencies'
+    required: false
+    default: true
+runs:
+  using: 'composite'
+  steps:
+    - name: Install Node.js
+      uses: actions/setup-node@v3
+      with:
+        node-version: 16
+
+    - uses: pnpm/action-setup@v2
+      name: Install pnpm
+      with:
+        version: 8
+        run_install: false
+
+    - name: Get pnpm store directory
+      shell: bash
+      run: |
+        echo "STORE_PATH=$(pnpm store path --silent)" >> $GITHUB_ENV
+
+    - uses: actions/cache@v3
```

Comment:
`actions/setup-node` supports a `pnpm` as a key for their `cache` option. If you are aware of this and this is still needed, 👍 but just wanted to check